apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -44,6 +44,36 @@ class DataVisualizer extends React.Component {
     this.setState({...INITIAL_STATE});
   };
 
+  createPivotTable = (records, rowName, columnName) => {
```

Comment:
Parameter type comments would be helpful to me here.  Do I have these right?
```suggestion
  /**
   * @param {Array.<Object>} records
   * @param {string} rowName
   * @param {string} columnName
   */
  createPivotTable = (records, rowName, columnName) => {
```